.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -14,7 +23,9 @@
 
 ## Steps to Reproduce (for bugs)
 <!--- Provide a link to a live example, or - -->
-<!--- Fork the CodePen to reproduce https://codepen.io/alphalpha/pen/oqKJgG ->
+<!--- Fork any of these code sandbox templates: ->
+<!---   *  Basic template: https://codesandbox.io/s/fixed-data-table-basic-example-cwds0 ->
+<!---   *  Advanced template: https://codesandbox.io/s/fixed-data-table-advanced-example-3gd9x ->
```

Comment:
Let's have a [basic](https://codesandbox.io/s/fixed-data-table-basic-example-cwds0) and an [advanced](https://codesandbox.io/s/fixed-data-table-advanced-example-3gd9x) example for the user's convenience.